src/sampler/sampler.jl

```diff
@@ -37,6 +45,12 @@ Sample one new OPF instance and modify `data` in-place.
 function Random.rand!(rng::AbstractRNG, s::SimpleOPFSampler, data::Dict)
     pd, qd = rand(rng, s.load_sampler)
     _set_loads!(data, pd, qd)
+
+    if !isnothing(s.reserve_sampler)
+        MRR, rmin, rmax = rand(rng, s.reserve_sampler)
+        _set_reserve!(data, MRR, rmin, rmax)
+    end
```

Comment:
Having a default sampler struct that sets reserves requirements and capacities to zero would remove the need for a `Nothing`ness check here